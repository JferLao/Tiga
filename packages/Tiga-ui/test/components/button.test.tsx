import TiButton from '../../src/components/button'
// import mountTest from '../../../../tools/shared/mountTest'
import React from 'react'
import { renderToString } from 'nerv-server'

describe('按钮测试', () => {
  it('size测试', () => {
    const component = renderToString(<TiButton size='normal'>按钮</TiButton>)
    expect(component).toMatchSnapshot()
  })
})
