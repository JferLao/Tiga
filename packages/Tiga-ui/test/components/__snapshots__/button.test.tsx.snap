// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`按钮测试 size测试 1`] = `undefined`;
